index.html

```diff
@@ -12,6 +12,57 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="navbar-container">
+        <img class="hamburger" src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/menu-hamburger.svg" width="3%" height="3%" alt="logo"><!--little logo missing-->
+        <img class="logo" src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/karma-logo.svg" width="3%" height="3%" alt="logo"><!--little logo missing-->
```

Comment:
It's best to stick to sizing images in your CSS file instead of inline. Remove the `width` and `height` attributes here and add these styles to the appropriate place in style.css

Reply:
Also, don't give absolute paths to your images: here, the path of your image goes all the way to the root of your personal computer's folder structure. Which means that when I try to load your page, these images don't load because my computer doesn't have folders called `home/dom`
To resolve this, use relative paths. In this case, `./img/karma-logo.svg` should work.

Reply:
Thank you Lucy, Done